LEGO1/mxbitmap.cpp

```diff
@@ -14,15 +14,75 @@ MxBitmap::MxBitmap()
 // OFFSET: LEGO1 0x100bca10
 MxBitmap::~MxBitmap()
 {
-  if (this->m_info != NULL) {
+  if (this->m_info != NULL)
```

Comment:
yeah we need a clang-format thing, i'll open a pr for it

Reply:
Should probably discuss this with @itsmattkc first if we want that and if yes, how it should be configured